agent/fw_drupal.c

```diff
@@ -742,19 +754,16 @@ NR_PHP_WRAPPER(nr_drupal_wrap_module_invoke_all_before) {
 
   hook_copy = nr_php_arg_get(1, NR_EXECUTE_ORIG_ARGS);
   if (nr_php_is_zval_non_empty_string(hook_copy)) {
-    nr_stack_push(&NRPRG(drupal_module_invoke_all_hooks),
-                  hook_copy);
-    nr_stack_push(&NRPRG(drupal_module_invoke_all_states),
-                  (void*)!NULL);
+    nr_stack_push(&NRPRG(drupal_module_invoke_all_hooks), hook_copy);
+    nr_stack_push(&NRPRG(drupal_module_invoke_all_states), (void*)!NULL);
```

Comment:
This is just the formatting change, right?

Reply:
yes